python/example_code/glue/scenario_getting_started_crawlers_and_jobs.py

```diff
@@ -0,0 +1,296 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+Overview
+
+Shows how to use the AWS SDK for Python (Boto3) with AWS Glue to:
+
+1. Create and run a crawler that crawls a public Amazon Simple Storage
+   Service (Amazon S3) bucket and generates a metadata database that describes the
+   CSV-formatted data it finds.
+2. List information about databases and tables in your AWS Glue Data Catalog.
+3. Create and run a job that extracts CSV data from the source Amazon S3 bucket,
+   transforms it by removing and renaming fields, and loads JSON-formatted output into
+   another Amazon S3 bucket.
+4. List information about job runs and view some of the transformed data.
+5. Delete all resources created by the demo.
+"""
+
+import argparse
+import io
+import json
+import logging
+from pprint import pprint
+import sys
+import time
+import uuid
+import boto3
+from boto3.exceptions import S3UploadFailedError
+from botocore.exceptions import ClientError
+
+from glue_wrapper import GlueWrapper
+from question import Question
+
+logger = logging.getLogger(__name__)
+
+
+# snippet-start:[python.example_code.glue.Scenario_GetStartedCrawlersJobs]
+class GlueCrawlerJobScenario:
+    """
+    Encapsulates a scenario that shows how to create an AWS Glue crawler and job and use
+    them to transform data from CSV to JSON format.
+    """
+    def __init__(self, glue_client, glue_service_role, glue_bucket):
+        """
+        :param glue_client: A Boto3 Glue client.
+        :param glue_service_role: An IAM role that Glue can assume to gain access to the
```

Comment:
IAM role --> AWS Identity and Access Management (IAM) role
Glue --> AWS Glue

Reply:
fixed